test/external-modules/multi-project/build.gradle

```
@@ -11,7 +11,7 @@ esplugin {
 
 dependencies {
   testImplementation project(path: ':test:test-clusters')
-  clusterModules project(':test:external-modules:test-multi-project')
+  clusterModules(project(':test:external-modules:test-multi-project'), project(":modules:ingest-common"))
```

I think the more common syntax is:
```suggestion
  clusterModules project(':test:external-modules:test-multi-project')
  clusterModules project(":modules:ingest-common")
```